docs/gmi.md

```diff
@@ -155,23 +118,25 @@ gmi.exit();
 
 should be used. This will return the user to the appropriate place.
 
-
-
 ## Data fields
 
 GMI exposes several read-only properties.
 
 #### gmi.embedVars
-Contains the JavaScript object that was entered as JSON in the 'Set-up Data'
-field in the GamesGrid CMS.
+This exposes any custom settings you want to save in our CMS and expose via GMI 
+i.e. if your game is multi-lingual your TPM can add a language param that can be
+accessed with:
+
+````
+gmi.embedVars.language
+````
```

Comment:
While this is a valid example, I worry that this may confuse devs, as in practice we're more likely to build separate localised GDZs. Can we think of a different configuration parameter to use as an example?